rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -556,13 +612,32 @@ def _create_model_data(
         # explicitly add last dimension to label_ids
         # to track correctly dynamic sequences
         model_data.add_features(LABEL_IDS, [np.expand_dims(label_ids, -1)])
-        model_data.add_features(TAG_IDS, [tag_ids])
+        for tag_name, tag_ids in crf_tag_ids.items():
+            model_data.add_features(f"{tag_name}_{TAG_IDS}", [tag_ids])
 
         model_data.add_mask(TEXT_MASK, TEXT_FEATURES)
         model_data.add_mask(LABEL_MASK, LABEL_FEATURES)
 
         return model_data
 
+    def _tag_ids_for_crf(self, example: Message, crf_layer: CRFLayer) -> np.ndarray:
+        """Create a np.array containing the tag ids of the given message for the given
```

Comment:
If I remember correctly, first line of a doc string should be one sentence followed by a blank line, then additional description